src/shogun/features/DotFeatures.cpp

```diff
@@ -345,10 +349,13 @@ void CDotFeatures::benchmark_dense_dot_range(int32_t repeats)
 	float64_t* w= SG_MALLOC(float64_t, d);
 	float64_t* out= SG_MALLOC(float64_t, num);
 	float64_t* alphas= SG_MALLOC(float64_t, num);
+#ifdef HAVE_LINALG_LIB
```

Comment:
ah noooo
This is something I really want to avoid ... double code.
Rather assume that linalg lib is available always.
@lambday @vigsterkr what about just requiring this? We had the same thing for eigen3 and it was a pain -- also lots of bugs in the manual code ...